dGame/dComponents/InventoryComponent.h

```diff
@@ -477,6 +477,8 @@ class InventoryComponent final : public Component {
 	 * @param document the xml doc to load from
 	 */
 	void UpdatePetXml(tinyxml2::XMLDocument& document);
+	void ManageVendorBuybackInventory(std::vector<LWOOBJID>& itemVector, Item* newItem, Inventory* inventory, bool removeItem);
```

Comment:
The first argument to this function can be removed.  the argument you set it as in every call is already a member of this.  You can already just access the container without passing it as a parameter.  newItem is never used as a pointer, only as an objectID.  Please convert this to an LWOOBJID instead.  removeitem as an argument is unused and should be removed.  inventory as a pointer is never used outside of the type.  Please convert this to just an eInventoryType instead.